From 1fe937835311a2d0afbc85ce4a4dbd3c9e0c912f Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Mon, 26 Nov 2018 11:19:45 +0800
Subject: [PATCH] Subject: [PATCH] Always use devmapper for kpartx

Do not try to compute the LIBDM_API_COOKIE make variable
from host information when cross-compiling.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Aws Ismail <aws.ismail@windriver.com>
Signed-off-by: Joe Slater <joe.slater@windriver.com>

Rebase to 0.7.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase to 0.7.7
Signed-off-by: Changqing Li <changqing.li@windriver.com>

Rebase to 0.7.9
Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 kpartx/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/kpartx/Makefile b/kpartx/Makefile
index 7eb467e..c143321 100644
--- a/kpartx/Makefile
+++ b/kpartx/Makefile
@@ -8,9 +8,7 @@ LDFLAGS += $(BIN_LDFLAGS)
 
 LIBDEPS += -ldevmapper
 
-ifneq ($(call check_func,dm_task_set_cookie,/usr/include/libdevmapper.h),0)
-	CFLAGS += -DLIBDM_API_COOKIE
-endif
+CFLAGS += -DLIBDM_API_COOKIE
 
 OBJS = bsd.o dos.o kpartx.o solaris.o unixware.o dasd.o sun.o \
 	gpt.o mac.o ps3.o crc32.o lopart.o xstrncpy.o devmapper.o
-- 
2.7.4

